### ISSUE TITLE
QA | Data Analytics v0.6.0.beta.1

### ISSUE THREAD

**Issue by EnriqueOrt-GSA** [GSA/px-benefit-finder#1660]
### Domain

data

### Related Issue(s)

[Create data layer event(s) to track error messages in analytics](https://github.com/GSA/px-benefit-finder/issues/1501)	https://github.com/GSA/px-benefit-finder/issues/1501

### Description

Data layer events QA 

Due by 08/12/2024 EOD or as soon as available

**Comment by EnriqueOrt-GSA** [comment]
@scottqueen-bixal @fongcindy @ErnestoFernandezAlvarez  for visibility

**Comment by EnriqueOrt-GSA** [comment]
@nickpistone - any updates to share on this QA work? Thanks in advance

**Comment by nickpistone** [comment]
I'm attaching a word doc detailing the tests I performed and the results I got. I found one issue. When I corrected some, but not all of the errors and resubmitted the form, the error event did not fire again. This is not a big deal as we have already tracked the initial error and it accurately tracks a successful form submission as well. So we are only missing repeated errors.

[Error Tracking QA.docx](https://github.com/user-attachments/files/16591609/Error.Tracking.QA.docx)
